db/migrate/20240326024112_add_manager_to_projects.rb

```diff
@@ -0,0 +1,6 @@
+class AddManagerToProjects < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :projects, :manager, foreign_key: { to_table: :users }
+    add_column :users, :is_project_manager, :boolean
```

Comment:
is this as a permission?  looks like a manger role already exists under user. should that be used?

looseends_development=# select role from users group by role;
  role   
---------
 manager
 admin
 user